app/models/user.rb

```diff
@@ -15,4 +15,8 @@ class User < ApplicationRecord
   def most_recent_posts
     posts.order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- [ ] [OPTIONAL] Great Job by implementing the `user.admin` method. Kudos 👍 . But I'll suggest you to add the boolean (true or false) so that we could know if that user has the role of admin or not.
```ruby
def admin?
     role == 'admin' ? true : false
end
```